src/contract/deploy.rs

```diff
@@ -51,12 +54,22 @@ impl<T: Transport> Builder<T> {
         let options = self.options;
         let eth = self.eth;
         let abi = self.abi;
+        let code = code.into();
+
+        let mut code_hex = String::from_utf8(code).unwrap();
+
+        for (lib, address) in self.linker {
+            let re = Regex::new(&format!("__{}_+", lib)).unwrap();
```

Comment:
Why do we need `Regex` for that? seems that it's a static replacement, no?

Reply:
If I understand correctly the amount of underscores is dynamic

Reply:
I think it's rather padded to be 32-bytes long.

Reply:
Makes sense, so you prefer that I'll use `len()`, `repeat()` and `replace()` instead of regex?
(and i'll look for some docs saying that's really the padding)

Reply:
@elichai either this or just use rust `format!` patterns to fill to desired width: https://doc.rust-lang.org/std/fmt/index.html#fillalignment